src/sentry/static/sentry/app/components/projectHeader/index.jsx

```diff
@@ -115,11 +115,20 @@ class ProjectHeader extends React.Component {
                 title={activeEnvironmentTitle}
                 className="environment-selector-toggle"
               >
-                <MenuItem onClick={clearActiveEnvironment}>
+                <MenuItem
+                  onClick={clearActiveEnvironment}
+                  className={activeEnvironment === null && 'active'}
+                >
                   {allEnvironmentsLabel}
                 </MenuItem>
                 {environments.map(env => (
-                  <MenuItem key={env.id} onClick={() => setActiveEnvironment(env)}>
+                  <MenuItem
+                    key={env.id}
+                    onClick={() => setActiveEnvironment(env)}
+                    className={
+                      activeEnvironment && activeEnvironment.name === env.name && 'active'
```

Comment:
what if activeEnvironment is ""? i forget how the weird cases work

Reply:
It's not an issue here